<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridSettingsWidget</class>
 <widget class="QWidget" name="GridSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="Coord2DInput" name="offsetInput" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lSpacing">
     <property name="text">
      <string>Spacing</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="colorBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorChooseButton">
       <property name="text">
        <string>â€¦</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="Coord2DInput" name="spacingInput" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="Coord2DInput" name="gridSizeInput" native="true"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lColor">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lGridSize">
     <property name="text">
      <string>Grid</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lOffset">
     <property name="text">
      <string>Offset</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lScale">
     <property name="text">
      <string>Scale</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="Coord2FInput" name="scaleInput" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Coord2DInput</class>
   <extends>QWidget</extends>
   <header>src/ui/coord2dinput.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Coord2FInput</class>
   <extends>QWidget</extends>
   <header>src/ui/coord2finput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
